docs/sources/contribute-documentation/contribute-release-notes/index.md

```diff
@@ -14,71 +14,129 @@ keywords:
+   - **DOCUMENTATION URL (OPTIONAL)** - URL to the public documentation for this feature.
+   - **GRAFANA URL PATH (OPTIONAL)** - URL path to the feature in a Grafana Cloud stack. For example, `/alerting/notifications`.
+   - **INTERNAL INFORMATION (OPTIONAL)** - Information for Grafana Labs employees only. For example, ProductDNA, slack channel, FAQ, training docs or videos. Used for training and internal announcements.
+   - **ENABLEMENT VIDEO (OPTIONAL)** - Link to the video used for enablement.
 
-### At the end of a release
+1. Click **Save**. The entry is now in **Draft** status.
+1. If your entry is ready to publish, proceed to step 4. If your entry requires review, follow these steps:
 
-Complete the following steps in the days leading up to the release:
+   1. In the **Status** drop-down, select **In review.**
 
-1. Five days before the What's new freeze, the technical program manager sends a last call to developer teams to get in their What's new content.
-1. The day after the freeze date, a PM reviews all content and adjusts wording, where appropriate.
-<!-- not sure who is doing this review -->
-1. Once the PM has completed their review, the technical writer completes a copy edit.
-1. When the copy edit is complete, the technical writer changes the PR status from **Draft** to **Ready for Review** to signal to other stakeholders that the PR is now ready for any further review. This should happen three to five days before the private/public release date to ensure time for further edits.
-1. The technical writer finalizes the What's new.
-1. The technical writer coordinates with Release Management and the GTM team for precise timing of when to merge the What's new doc into `main`.
-1. On release day (private release) or before release day (public), the technical writer and merges the What's new branch into `main`. If `main` is no longer the same release as the upcoming release, the technical writer should add the appropriate backport label to the PR.
+      A review PR is generated in the `grafana/website` repository. **The Documentation Team does not automatically review these requests; teams that create What’s new entries are responsible for determining their own review process.** However, there are two weekly Office Hours meetings offered by the Documentation Team that you’re welcome to attend for guidance and assistance:
 
-The What's new is published in the "next" docs.
+      - [Docs squad office hours (early)](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NmoxZ2w0NHBoazgxaW80cTN0MW82aG1xMmxfMjAyMzExMjJUMDkwMDAwWiBpc2FiZWwubWF0d2F3YW5hQGdyYWZhbmEuY29t&tmsrc=isabel.matwawana%40grafana.com&scp=ALL)
+      - [Docs squad office hours (late)](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NTI0MWZxYTYwY3FjYW5waWM1OWVjMDBkaWpfMjAyMzExMjJUMjEwMDAwWiBpc2FiZWwubWF0d2F3YW5hQGdyYWZhbmEuY29t&tmsrc=isabel.matwawana%40grafana.com&scp=ALL)
+
```

Comment:
```suggestion
      - [Docs squad office hours (early)](https://docs.google.com/document/d/1QaH9PwIZ_6-6Udhmy7Zhwme72ZZLqSTiX8HYFFi6AE4/edit#heading=h.9o53ccbx7xrw)
      - [Docs squad office hours (late)](https://docs.google.com/document/d/12XK3XYEZWU3uIPluq3sn5HSGqWHBHJkktqlyQZHj_Uw/edit#heading=h.9o53ccbx7xrw)

```
I think the links you provided @imatwawana will just link folks to their own calendar. Replaced with links to the agendas

Reply:
Thanks for the catch. I noticed the links didn't look quite right later but forgot about it.